deltachat-ios/Info.plist

```diff
@@ -39,13 +39,13 @@
 	<key>LSSupportsOpeningDocumentsInPlace</key>
 	<true/>
 	<key>NSCameraUsageDescription</key>
-	<string>Allowing access to the camera lets you take photos and videos.</string>
+	<string>This is used to take photos and videos during chats or scan QR codes.</string>
 	<key>NSContactsUsageDescription</key>
-	<string>Allowing access to your address book lets you chat with contacts from your device.</string>
+	<string>This is used to import email addresses of your contacts.</string>
 	<key>NSMicrophoneUsageDescription</key>
-	<string>Allowing access to the microphone lets you record audio.</string>
+	<string>This is used to record voice messages during chats.</string>
```

Comment:
i find this "during chats" wording a bit strage. may be regarded as if this always happens "during chatting".
maybe better sth. as "Delta Chat needs access to your microphone to record voice messages you can send to your chat contacts."

Reply:
But aren't we using the microphone to send voice messages only in chats?

Reply:
sure, but this "during chats" wording sounds strange to me - and may sound strange to reviewers as well. maybe i am wrong. 

for my suggestion, i peeked on signal wordings, btw, i think this is not the worse idea as they make the review process at some point. maybe just take the sentences from https://github.com/signalapp/Signal-iOS/blob/master/Signal/Signal-Info.plist#L116 and adapt them to our usage.

Reply:
I have written explanations that are more general, but more understandable by the user. We can change these comments as we release new features. For now, at least I couldn't find anything better for the microphone.